File: packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/ImageManipulatorPackage.java
package expo.modules.imagemanipulator;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.core.BasePackage;
import expo.core.ExportedModule;

public class ImageManipulatorPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ImageManipulatorModule(context));
  }
}

File: android/versioned-abis/expoview-abi45_0_0/src/main/java/abi45_0_0/expo/modules/imagemanipulator/ImageManipulatorPackage.kt
package abi45_0_0.expo.modules.imagemanipulator

import android.content.Context
import abi45_0_0.expo.modules.core.BasePackage

class ImageManipulatorPackage : BasePackage() {
  override fun createExportedModules(context: Context) = listOf(ImageManipulatorModule(context))
}
